src/main/java/com/myhome/controllers/response/GetAmenityBookingsResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.myhome.controllers.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+public class GetAmenityBookingsResponse {
+    String amenityId;
```

Comment:
That fields could be private and class annotated as @Data

Reply:
done